src/acknowledgments.md

```diff
@@ -68,21 +117,20 @@ Thanks to everyone who opened a Pull Request and wrote a story, shiny future or
 * [cryptoquick](https://github.com/cryptoquick)
 * [Darksonn](https://github.com/Darksonn)
 * [doc-jones](https://github.com/doc-jones)
-* [ehuss](https://github.com/ehuss)
```

Comment:
Just doing a super quick skim of this diff... this removal surprises me a little bit, since ehuss has two merged PRs

Reply:
I didn't even notice some people were removed, whoops :sweat_smile:

Reply:
I didn't notice that either!